blacklist

```diff
@@ -142,6 +142,7 @@
   "dreed47/redfin",
   "DSorlov/hasl-platform",
   "dummylabs/watchman",
+  "DurgNomis-drol/ha_toyota",
```

Comment:
Only add new things in this PR.
If a repository needs to be removed, use the remove repository issue template in hacs/integraiton.

Reply:
fixed, removal is here https://github.com/hacs/integration/issues/4521